src/lib/theme.ts

```diff
@@ -10,8 +10,9 @@ export const styles = {
             transition duration-200`,
         sm: `px-3 py-1.5 text-sm`,
         md: `px-4 py-2 text-base`,
+        highlighted: `bg-gray-700/80 border-gray-500`,
         trigger: `border-l border-[inherit]`,
-        control: `border-l border-[inherit] hover:bg-gray-700/95 hover:border-gray-600
+        control: `border-[inherit] hover:bg-gray-700/95 hover:border-gray-600
```

Comment:
Removed `border-l` here as it would put a border in the middle but also on the left of the first button
Added the `styles.button.trigger` to the search bar button to have that `border-l` back. I wonder if it would be better to just add `border-l` instead?